clusterloader2/pkg/measurement/common/simple/profile.go

```diff
@@ -102,6 +103,11 @@ func gatherProfile(componentName, profileKind, host, provider string) ([]measure
 	getCommand := fmt.Sprintf("curl -s localhost:%v/debug/pprof/%s", profilePort, profileKind)
 	sshResult, err := measurementutil.SSH(getCommand, host+":22", provider)
 	if err != nil {
+		if provider == "gke" {
+			// Only logging error for gke. SHHing to gke master is not a standard case.
```

Comment:
s/is not a standard case/is not supported/

Reply:
Done.